exporter/opentelemetry-exporter-otlp-proto-grpc/tests/logs/test_otlp_logs_exporter.py

```diff
@@ -232,6 +264,31 @@ def test_env_variables(self, mock_exporter_mixin):
         self.assertIsNotNone(kwargs["credentials"])
         self.assertIsInstance(kwargs["credentials"], ChannelCredentials)
 
+    @patch.dict(
+        "os.environ",
+        {
+            OTEL_EXPORTER_OTLP_LOGS_ENDPOINT: "logs:4317",
+            OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE: THIS_DIR
+            + "/../fixtures/test.cert",
+            OTEL_EXPORTER_OTLP_LOGS_HEADERS: " key1=value1,KEY2 = VALUE=2",
+            OTEL_EXPORTER_OTLP_LOGS_TIMEOUT: "10",
+            OTEL_EXPORTER_OTLP_LOGS_COMPRESSION: "gzip",
+        },
+    )
+    @patch(
+        "opentelemetry.exporter.otlp.proto.grpc.exporter.OTLPExporterMixin.__init__"
+    )
+    def test_env_variables_with_only_certificate(self, mock_exporter_mixin):
```

Comment:
It looks like we now log a couple of errors when the client certs are not provided. Should we remove them? I'm thinking that we should assert here that we're not doing any additional logging.

Reply:
I think ideally they should only log an exception if the user has configured certs but don't provide valid paths. So actually `load_credentials` should accept `Optional[str]` params and only perform `read_file` if the params are not `None.

Reply:
Resolved, tyvm :)